core/crypto.mk

```diff
@@ -124,17 +124,57 @@ $(eval $(call cryp-dep-one, AES, ECB CBC CTR CTS XTS))
 # If no DES cipher mode is left, disable DES
 $(eval $(call cryp-dep-one, DES, ECB CBC))
 
+###############################################################
+# libtomcrypt (LTC) specifics, phase #1
+# LTC is only configured via _CFG_CORE_LTC_ prefixed variables
+###############################################################
+
+# If LTC is the cryptolib, pull configuration from CFG_CRYPTO_xxx
+ifeq ($(CFG_CRYPTOLIB_NAME),tomcrypt)
 # dsa_make_params() needs all three SHA-2 algorithms.
 # Disable DSA if any is missing.
 $(eval $(call cryp-dep-all, DSA, SHA256 SHA384 SHA512))
 
-cryp-one-enabled = $(call cfg-one-enabled,$(foreach v,$(1),CFG_CRYPTO_$(v)))
-cryp-all-enabled = $(call cfg-all-enabled,$(foreach v,$(1),CFG_CRYPTO_$(v)))
+# Assign _CFG_CORE_LTC_xxx based on CFG_CRYPTO_xxx
```

Comment:
Nit: "Assign _CFG_CORE_LTC_xxx based on CFG_CRYPTO_yyy" maybe? (since there isn't a one-to-one mapping).

Reply:
I'll fix